blog/2019-09-ngrx-data-views/README.md

```diff
@@ -0,0 +1,419 @@
+---
+title: "Handling normalized entities with data views"
```

Comment:
Add "NgRx" context to title. We should discuss the title in general as it is THE entrypoint for every user

Reply:
ok, my creativity is taking me as far as `currentTitle + ' with NgRx'`. Suggestions?

Reply:
`Enterprise Business Objects with NgRx`?

Reply:
`Handling Data with NgRx in Enterprise Scenarios`?

Reply:
`Getting the most out of NgRx selectors`? -- maybe too populist

Reply:
`NgRx Selectors in Real-World Problems/Scenarios`

Reply:
`NgRx Selectors in Real-World Problems/Scenarios`

Reply:
Hmm... the more I think about it the more I like "normalized entities" since it exactly decribes what the post is about. If it comes to me, this is fine:
> Handling normalized entities in NgRx

Opinions? @JohannesHoppe

Reply:
### NgRx Data Views: How to normalize entities for large enterprise applications

Reply:
Sounds good! I'm not sure whether the term "data views" needs to be mentioned in the title. Having "enterprise" in the title is good, but I wouldn't mention "large" here

Reply:
i try to catch many bullsh*t bingo words as possible.

i like the word "ngrx data views" very much. This describes the pattern perfectly and could be a widely used term in future! :-)

Reply:
Okay, agree! We could establish this term by actively using it.


> NgRx Data Views: How to normalize entities for enterprise applications

Reply:
Okay, I agree with the first part. Good Idea :+1: 
But isn't it actually `How to de-normalize entities for enterprise applications`?

Reply:
That's correct.

## NgRx Data Views: How to de-normalize entities for large enterprise applications

Reply:
Yep. I'd still go without the "large" to unclutter he title a bit. But I'd leave this decision to @dhhyi ;-)

Reply:
I think the "large" adds another layer, both in meaning (enterprise, hehe) as in the tone sequence when I hear the title. I'll go with it. Thanks ;)